src/main/java/com/volasoftware/tinder/services/implementations/AuthenticationServiceImpl.java

```diff
@@ -26,111 +27,130 @@
 @RequiredArgsConstructor
 public class AuthenticationServiceImpl implements AuthenticationService {
 
-    private final PasswordEncoder passwordEncoder;
+  private final PasswordEncoder passwordEncoder;
 
-    private final JwtService jwtService;
+  private final JwtService jwtService;
 
-    private final VerificationTokenService verificationTokenService;
+  private final VerificationTokenService verificationTokenService;
 
-    private final EmailService emailService;
+  private final EmailService emailService;
 
-    private final EmailValidator emailValidator;
+  private final EmailValidator emailValidator;
 
-    private final AccountRepository accountRepository;
+  private final AccountRepository accountRepository;
 
-    private final AccountMapper accountMapper;
+  private final AccountMapper accountMapper;
 
-    private final FileService fileService;
+  private final FileService fileService;
 
-    private final AuthenticationManager authenticationManager;
+  private final AuthenticationManager authenticationManager;
 
-    @Value("${server.base-url}")
-    private String baseUrl;
+  @Value("${server.base-url}")
+  private String baseUrl;
 
-    @Value("${server.verify-url}")
-    private String verifyUrl;
+  @Value("${server.verify-url}")
+  private String verifyUrl;
 
-    @Override
-    public List<AccountDto> getAll() {
-        List<Account> accounts = accountRepository.findAll();
+  @Override
+  public List<AccountDto> getAll() {
+    List<Account> accounts = accountRepository.findAll();
```

Comment:
Its a good idea to use Pageable object inside findAll method, suppose you have millions of accounts, this line can easily exhaust the database.

Reply:
Thanks for the suggested object. I`ll read more about it, before implementing it.

Reply:
The repository now returns results base on passed Pageable object.